assets/css.css

```diff
@@ -0,0 +1,79 @@
+
+  textarea {
+    align-content: center;
+    width: 1200px;
+    height: 150px;
+    padding: 12px 20px;
+    box-sizing: border-box;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    background-color: #f8f8f8;
+    font-size: 16px;
+    resize: none;
+  }
+    .button {
+        background-color:#0CB3EE;
+        border: none;
+        color: white;
+        float:right;
+        padding: 15px 32px;
+        text-align: center;
+        text-decoration: none;
+        font-size: 16px;
+        margin: 4px 2px;
+        cursor: pointer;
+        transition-duration: 0.4s;
+        box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+        }
+
+       /* Add a black background color to the top navigation */
+.topnav {
+    background-color:white;
+    overflow: hidden;
+   
+}/* Style the links inside the navigation bar */
+.topnav a {
+    float: left;
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 18px 20px;
+    text-decoration: none;
+    font-size: 17px;
+}
+/* Change the color of links on hover */
+.topnav a:hover {
+    background-color: #0CB3EE;
+    color: black;
+    
+}
+table {
+
+    border-collapse: collapse;
+    width: 1200px;
+    display: center;
+    height: 150px;
+}
+
+th, td {
+    text-align:center;
+    padding: 8px;
+}
+
+tr:nth-child(even){background-color: #f2f2f2}
+
+th {
+    background-color:  #0CB3EE;
+    color: white;
+}
+
+.formdimension {
+    width: 100%;
+    height: 300 px;
```

Comment:
Just checking whether the space between measurement and unit doesn't affect the display.